contracts/requirements.txt

```diff
@@ -1,3 +1,5 @@
+gitpython==2.1.11
```

Comment:
this is `requirements.txt` to use for contract compiling. Not sure if these should be mixed.